032-MLOpsFromScratch/Student/02-UnitTesting.md

```diff
@@ -0,0 +1,41 @@
+# Challenge 2 – Create a Unit Test in Azure DevOps
+
+[< Previous Challenge](./01-TimeSeriesForecasting.md) - **[Home](../README.md)** - [Next Challenge >](./03-BuildPipeline.md)
+
+## Introduction
+
+Use this challenge to perform tasks to confirm data has been extracted from
+source and saved in the working directory
+
+## Description

```

Comment:
I think you have too much redundancy in this challenge text.

The assumption is that a student will have already done Challenge 1 and done it properly before they ever get here. I would remove the text that explains what they should've done.

Even if your Challenge 2 text is very little, that's fine.. it's the success criteria that matter.